2026.01

Geenz opened the pull request:
as of = 2026-03-05 for RC deploy
build = https://github.com/secondlife/viewer/releases/tag/Second_Life_Release%23e572093e-2026.01
cohort = 2026.01
deployed = https://github.com/secondlife/viewer/releases/tag/Second_Life_Release%23e572093e-2026.01
desired = 3000
relnotes:

## **Second Life Viewer 2026.01 — Legacy Search & Stability**

The **Second Life Viewer 2026.01** release focuses on stability, usability, and a few very welcome quality-of-life updates. The headline change is the return of **Legacy Search**, (found in the World menu) alongside improvements to camera controls, snapshots, gestures, and a wide range of crash and reliability fixes across inventory, rendering, uploads, and voice.

As usual, this release also includes a substantial amount of behind-the-scenes work aimed at making the viewer feel smoother and more resilient in everyday use.

---

## **New features**

* **Legacy Search has returned**, It is not the default search and can be found in the World menu, restoring the classic search experience many residents rely on.  
* Added the ability to **hide transparent reflection probes**, with a new menu option and a default shortcut: **Alt \+ Shift \+ T**.  
* The **Resident Chooser** now supports searching by **Agent ID**, improving support and administrative workflows.

---

## **Quality-of-life improvements**

* The **Gestures** window now includes a **sortable “Status” column**, making it easier to see which gestures are active at a glance.  
* **Snapshot to Inventory** adds **1024×1024** as a selectable resolution.  
* **Camera controls** received multiple improvements, including better aspect-ratio handling and a shortcut for panning the camera downward.  
* Improved **outfit list performance**, especially for larger wardrobes.  
* Better handling of **external editors**, with improved fallback to the operating system’s default app when no editor is configured.  
* Restored support for **avatar cloth**.

---

## **Bug fixes and stability improvements**

This release includes a broad set of fixes across the viewer:

### **Inventory and UI**

* Fixed a long-standing issue that could cause the viewer to **freeze while fetching inventory**.  
* Resolved multiple issues related to **Copy SLURL**, including obsolete link formats and Favorites bar behavior.  
* Fixed cases where **@mentions could appear twice** when using certain characters.  
* Corrected the **“Don’t show me in search”** option so its visual state updates properly.  
* Fixed an issue where the cache could be cleared on every launch if the cache location was invalid.

### **Graphics and rendering**

* Multiple crash fixes in rendering, texture handling, and LOD-related code paths.  
* Fixed crashes related to shader setup and attachment edge cases.  
* Improvements to GPU benchmarking, including better defaults on **Apple Silicon** systems.

### **Uploads**

* Several fixes for **glTF uploads**, including crash fixes and issues selecting textures during upload.

### **Voice (WebRTC)**

* Fixed crashes during **WebRTC initialization**.  
* Improvements to voice reconnection behavior after disconnects.  
* Adjustments to device-switching logic to prevent instability.
* Fixed support for Bluetooth headsets disconnecting

### **Watchdog**

* The **Watchdog** system has been re-added but is currently in a disabled state.  We will re-enable this in a later release.

---

## **Try it out and tell us what you think**

The Second Life Viewer 2026.01 is now available to download from the official Second Life website. As always, we encourage you to try it out and let us know how it performs in your day-to-day use.

If you run into issues, please file bugs via **Canny (Feedback)** so they can be tracked and investigated by the team. Your reports directly help shape future releases.

Thanks for helping us make the Second Life Viewer better — and we’ll see you in the next release!

LLGuru committed:
#2997 The 'Reset Skeleton' option is missing in the right-click menu

akleshchev committed:
#2639 Reapplied GPU benchmark fix

First test returns quarter to a half the throughput, do two tests.
May be caused by driver, may be some 'energy saving', but not important
enough to spend time investingating. It was working the same way prior
to ExtraFPS, but viewer was running an extra CPU test that 'preheated'
the system.

Also increasing minimum throughput as numerous new features, like
mirrors and pbr were added and requirements are now higher.

dependabot[bot] committed:
Bump actions/stale from 6 to 10

Bumps [actions/stale](https://github.com/actions/stale) from 6 to 10.
- [Release notes](https://github.com/actions/stale/releases)
- [Changelog](https://github.com/actions/stale/blob/main/CHANGELOG.md)
- [Commits](https://github.com/actions/stale/compare/v6...v10)

---
updated-dependencies:
- dependency-name: actions/stale
  dependency-version: '10'
  dependency-type: direct:production
  update-type: version-update:semver-major
...

Signed-off-by: dependabot[bot] <support@github.com>

maxim-productengine committed:
#4411 WIP initial restoring of Legacy Search

akleshchev committed:
#4652 Improve switching of webrtc devices

Hecklezz committed:
Updated APR library and removed hotfix patch in APR cmake

marchcat committed:
Ensure final EOF in LLProcess test (#4877)

akleshchev committed:
#4876 Fix material not applying

1. Dupplicate call in llselectmgr can cause race conditions
2. lltooldragndrop wasn't updating server in one case

akleshchev committed:
p#494 Add search by agent ID to resident chooser

maxim-productengine committed:
#4411 WIP add Web tab to legacy Search

akleshchev committed:
#4884 Crash at resetRegionCrossingTimer

akleshchev committed:
#4886 Crash on LLTeleportRequestViaLure::toOstream

maxim-productengine committed:
#4411 fix additional search options - category and maturity

marchcat committed:
#4887 Fix access violation in LLInventoryModelBackgroundFetch::bulkFetchViaAis

akleshchev committed:
Revert "Improve switching of webrtc devices"

Wrong approach. Might need to split workerDeployDevices into
separate recording and rendering variants.
This partiall reverts commit bb26aa3c2cb8ff961668cf0ad8180d3e9c57f941,
I left log lines.

maxim-productengine committed:
#4411 UI adjustments

maxim-productengine committed:
Fix indentation and trailing-whitespace

akleshchev committed:
#4879 An Emoji in a display name can affect text in chat

An emoji was spliting inlineview thus affecting displayed text.

maxim-productengine committed:
Mac build fix and clean up

akleshchev committed:
#3649 Incorrect camera aspec ratio

Everything else uses width/height ratio

akleshchev committed:
#3670 Add a sortable status column to the list of gestures

akleshchev committed:
#1112 Add 1024x1024 size to Snapshot to Inventory

maxim-productengine committed:
Merge pull request #4900 from secondlife/maxim/2025.08-legacy-search

Restore Legacy Search

akleshchev committed:
SL-3609 Fix camera POV jump when avatar crosses a region boundary

sFlycamPosition was using local coordinates instead of global ones thus
was not tracking global changes and was force setting local position to
an obsolete value.

maxim-productengine committed:
#4411 fix occurrences of fake BOOL

dependabot[bot] committed:
Bump actions/github-script from 7 to 8 (#4865)

Bumps [actions/github-script](https://github.com/actions/github-script) from 7 to 8.
- [Release notes](https://github.com/actions/github-script/releases)
- [Commits](https://github.com/actions/github-script/compare/v7...v8)

---
updated-dependencies:
- dependency-name: actions/github-script
  dependency-version: '8'
  dependency-type: direct:production
  update-type: version-update:semver-major
...

Signed-off-by: dependabot[bot] <support@github.com>
Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

dependabot[bot] committed:
Bump actions/labeler from 4 to 6 (#4866)

Bumps [actions/labeler](https://github.com/actions/labeler) from 4 to 6.
- [Release notes](https://github.com/actions/labeler/releases)
- [Commits](https://github.com/actions/labeler/compare/v4...v6)

---
updated-dependencies:
- dependency-name: actions/labeler
  dependency-version: '6'
  dependency-type: direct:production
  update-type: version-update:semver-major
...

Signed-off-by: dependabot[bot] <support@github.com>
Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

Commit:
Refactor labeler.yaml to use changed-files syntax

dependabot[bot] committed:
Bump actions/setup-python from 4 to 6 (#4922)

Bumps [actions/setup-python](https://github.com/actions/setup-python) from 4 to 6.
- [Release notes](https://github.com/actions/setup-python/releases)
- [Commits](https://github.com/actions/setup-python/compare/v4...v6)

---
updated-dependencies:
- dependency-name: actions/setup-python
  dependency-version: '6'
  dependency-type: direct:production
  update-type: version-update:semver-major
...

Signed-off-by: dependabot[bot] <support@github.com>
Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

Commit:
Add bot* to allowlist in CLA workflow

akleshchev committed:
#4931 Fix name cache callbacks not having connections

akleshchev committed:
#4946 Crash at setShaders

Fix wrong delimeter and log if file isn't present

akleshchev committed:
Check what state pipe is in for a failed test (#4877)

maxim-productengine committed:
#4932 fix @mention being displayed twice if the name has a TM symbol

akleshchev committed:
#4955 Crash on itemChanged

maxim-productengine committed:
#4932 fix @mention being displayed twice if the name has a TM symbol

Geenz committed:
Remove cpuid feature bits verification in llprocessor.cpp (#4959)

* Remove cpuid feature bits verification in llprocessor.cpp

maxim-productengine committed:
#4907 use shortcut to pan camera down

akleshchev committed:
#4671 Fix impractically long am/pm labels

RyeMutt committed:
Restore support for wavy avatar cloth(#4963)

Signed-off-by: Rye <rye@alchemyviewer.org>

Hecklezz committed:
Fix crash in updateImagesCreateTextures accessing potentially freed image pointer

Signed-off-by: Hecklezz <tj8@live.com.au>

Hecklezz committed:
Fix crash in updateDistance updating LOD on null mVObjp

Signed-off-by: Hecklezz <tj8@live.com.au>

Hecklezz committed:
Fix crash in handleHover accessing null parent

Signed-off-by: Hecklezz <tj8@live.com.au>

akleshchev committed:
#4939 Bugsplat update

to version 1.2.6

Hecklezz committed:
Follow-up null checks on mVObjp to prevent crashes

Signed-off-by: Hecklezz <tj8@live.com.au>

akleshchev committed:
#4980 Crashes when uploading a glTF model

Functions used in Image::prep aren't thread safe, pass them to main
thread and wait for a result

fmartian committed:
Fix error when adding a widget filename to the mFileNames stack

base_filename is already an absolute path that was resolved in ffindSkinnedFilenames() above but the oushFileName() method tries to do the same again, resulting in an invalid path (the skin directory path is prepended twice)

Hecklezz committed:
Fix the cache from being cleared every launch if NewCacheLocation is invalid

Signed-off-by: Hecklezz <tj8@live.com.au>

akleshchev committed:
#4948 Crash at webrtc's initializeConnection

akleshchev committed:
#4939 Bugsplat often confuses log files on mac, try to make it more reliable

akleshchev committed:
#4951 Update bugsplat symbol upload

akleshchev committed:
p#505 GLTF model upload fails to pick textures

maxim-productengine committed:
#5000 clear duplicate shortcut used to activate Sunrise

akleshchev committed:
#4991 Cant set cut off mask

maxim-productengine committed:
#4735 Remove the word "Viewer" from application shortcut

akleshchev committed:
#4651 Improve session shutdown detection

akleshchev committed:
SL-9502 p#476 Fix visible hidden column

akleshchev committed:
#5026 Upload To Folder feature not working

akleshchev committed:
#5012 Switch 'chat' from untrusted, to click only

This command shouldn't be allowed from outside the app, but clicks should be valid

Hecklezz committed:
Fixed LLWindowMacOSX::setCursorPosition scaling gl_pos when it shouldn't

Signed-off-by: Hecklezz <tj8@live.com.au>

akleshchev committed:
MacOS build fix

akleshchev committed:
#5035 Fix missed am/pm switch

RyeMutt committed:
Introduce menu option and key bind(alt+shift+t) to show/hide reflection probes in transparency debug view

akleshchev committed:
#1963 Restore pelvis rotation debug settings

akleshchev committed:
#5046 Remove redundant updates in outfit list #1

akleshchev committed:
#5046 Fix accordion control's excessive rearranges #2

Since arrange is no longer part of LLInventoryItemsList::doIdle(),
reduced time limit.

akleshchev committed:
#5046 Optimize outfit's onRefreshComplete #3

RyeMutt committed:
Fix tabs in various source files introduced as part of legacy search

Signed-off-by: Rye <rye@alchemyviewer.org>

akleshchev committed:
#1937 The "Don't show me in search" option is not visually updated

akleshchev committed:
#4322 Upload>Model button is sometimes disabled on MacOS

RyeMutt committed:
#5072 Use make_shared for more efficient ref counting and allocation

RyeMutt committed:
Fix occasional build failure observed when llphysicsextensionsos fails to build before secondlife-bin on macos with havok

Signed-off-by: Rye <rye@alchemyviewer.org>

RyeMutt committed:
Replace boost noncopyable and undefined function usage with modern c++ deleted operator and constructor

RyeMutt committed:
#5078 Replace boost ptr_vector<nullable> with std::vector<std::unique_ptr>

Signed-off-by: Rye <rye@alchemyviewer.org>

RyeMutt committed:
#5078 Replace boost::hof::invocable with c++17 std::is_invocable

Signed-off-by: Rye <rye@alchemyviewer.org>

RyeMutt committed:
#5078 Remove dead llblowfish and llmail code plus related tests

Signed-off-by: Rye <rye@alchemyviewer.org>

RyeMutt committed:
#5078 Replace boost::function with std::function

* Replace boost::function usage with std::function for easier debugging and reduced compiler warnings
* Remove a few remaining instances of boost::noncopyable that were missed in tests

Signed-off-by: Rye <rye@alchemyviewer.org>

RyeMutt committed:
#5078 Replace boost::assign usage with modern c++ brace initialization

Signed-off-by: Rye <rye@alchemyviewer.org>

RyeMutt committed:
Fix build failure introduced in boost::function cleanup from missing bind header

RyeMutt committed:
#5078 Remove dead experimental atmospherics code

Remove dead experimental atmospherics code left over from EEP development

RyeMutt committed:
#5083 Fallback to OS open handler in external editor when no editor is explicitly set (#5077)

* Fallback to OS open handler in external editor when no editor is explicitly set

Signed-off-by: Rye <rye@alchemyviewer.org>

* Apply suggestion to use SystemRoot for windows OS root path in open handler

Co-authored-by: Ansariel Hiller <Ansariel@users.noreply.github.com>

---------

Signed-off-by: Rye <rye@alchemyviewer.org>
Co-authored-by: Ansariel Hiller <Ansariel@users.noreply.github.com>

akleshchev committed:
#5078 MacOS Build fix

RyeMutt committed:
Fix alpha debug overlay rendering in probes and breaking scene lighting (#5058)

Signed-off-by: Rye <rye@alchemyviewer.org>

akleshchev committed:
#5084 Ressurect Watchdog

akleshchev committed:
#5086 Fix 'Copy SLURL' generating obsolete http link

use https

akleshchev committed:
#5084 Better labeling of uncaught events, freezes, crashes

At least while watchdog is enabled it's more fair to label these as unknowns

akleshchev committed:
Switch version to 2026.01.0 after promotion of 2025.08

Geenz committed:
Merge pull request #5094 from secondlife/main

2025.08 -> Develop

Geenz committed:
Merge pull request #5096 from secondlife/develop

Switch version to 2026.01.0 after promotion of 2025.08

marchcat committed:
Build fix

Geenz committed:
Merge pull request #5100 from secondlife/develop

Develop build fixes to 2026.01

akleshchev committed:
#3612 "Copy SLURL" from Favorites bar not working

If region isn't online, can't get region position and form slurl

akleshchev committed:
#5109 LLExperienceCache crashes on a coroutine

akleshchev committed:
#4931 Fix missed name cache connection #2

akleshchev committed:
#3612 "Copy SLURL" from Favorites bar not working #2

akleshchev committed:
#4604 Tweak LLImageBase's 'out of memory' handling

Hecklezz commented:
Edit: Fixed in this merged PR to the Linux branch- https://github.com/secondlife/viewer/pull/5156

macOS compilation with Firestorm using XCode 26 fails due to the use of a fake bool that is not defined-
https://github.com/secondlife/viewer/blob/49c73ac7297ec7760a00bf36fa6339f2d0be1f95/indra/newview/llpaneldirland.cpp#L200 "llpaneldirland.cpp:200:9: error: unknown type name 'BOOL'"

This may not be an issue with Second Life viewer and it may compile fine I haven't tested, but probably best to remove the fake bool as there has already been a commit to do so in this same PR.

maxim-productengine committed:
#5122 update version format

maxim-productengine committed:
#5122 update version format #2

akleshchev committed:
#1112 Fix wrong option having size controls

in Snapshot to Inventory

akleshchev committed:
#5084 Improve watchdog's feedback

akleshchev committed:
#5084 Cover window's thread with watchdog

akleshchev committed:
#3612 Log issues with landmarks and rethrown exceptions

akleshchev committed:
#3612 Handle missing capabilities instead of blocking further downloads

Landmarks that failed due to missing caps were blocked from being
rerequested.

akleshchev committed:
#3612 Don't immediately fail download on startup, wait for the region

It's probably smarter to make asset storage wait with requests in
general, but that would require a stronger separation from cache which
is functional already and can be loading assets, so I'm pausing
individual requests instead, there is a maximum of 12, so there should
be no 'bloat' from waiting coros.

maxim-productengine committed:
revert #4735 Remove the word "Viewer" from application shortcut

maxim-productengine committed:
#5220 do not save 'Highlight Transparent Probes' setting between sessions

akleshchev committed:
#5232 Crash on mHoverItem in context menu

akleshchev committed:
#4991 Fix showing incorrect alpha

Due to _isAlpha alpha was considered as blend even if only one texture
had an alpha and was set to anything but blend.

akleshchev committed:
#5232 Crash in handleMessage when trying to access llsd's content

Crash at LLSD::Impl::assign

RyeMutt committed:
secondlife/viewer#5083 Fix external editor default open handling failing to work on mac and windows

akleshchev committed:
#3612 "Copy SLURL" from Favorites bar not working #3

akleshchev committed:
#5086 Update links from settings to https

akleshchev committed:
#5275 Optimize hasParcelLandmark

Implemented result caching.
hasParcelLandmark can get repeatedly called when performing operations
on landmarks en masse, which was causing repeated inventory searches
and leads to stalls with large collections of landmarks.

akleshchev committed:
#5275 Make landmark search by position cheaper

by stopping after first item was found.

akleshchev committed:
#4267 Slight logging improvement for mute list

akleshchev committed:
#5084 Adjust watchdog to avoid false-positives

And increase allowed time to be more in tune with disconnects.

maxim-productengine committed:
#5304 fix UI inconsistencies related to maturity setting in Legacy Search floater

akleshchev committed:
#5086 Restore guidebook to use http

roxanneskelly committed:
Fix failure to reconnect after disconnect and occasional dropout issue (#5322)

* Fix failure to reconnect after disconnect and occasional dropout issue

We were occasionally seeing dropouts which may have been caused by ICE
renegotiate requests.  The code is there to reconnect in that case,
but there were a few bugs, some of which were likely due to the webrtc upgrade.

Also, we were seeing failures to reconnect after voice server restart.

There were some issues with the PTT button that came up after the above issue was fixed.

* Added a clarification as part of CR

* We need to set mute state for p2p/adhoc/group calls as well

roxanneskelly commented:
I have read the CLA Document and I hereby sign the CLA

akleshchev committed:
#5046 Fix a long freeze when fetching inventory

akleshchev committed:
#2639 Fix benchmarked render level being too high for Apple Silicon

After benchmark fixes, render level jumped from lowest to high+, cap it at med+ until shadows get fixed to have less impact

marchcat committed:
#2997 Fix Reset Skeleton not working on animesh objects

Dan-Linden commented:
| 2026.01 Beta 26.1.0.21522948608 | Result | Comment |
|---|---|---|
| secondlife/iqa#609 Acceptance, Integration & Additional Tests | Passed | Passed |
| **Overall Result** | **Passed** | **OK to ship** |

**NOTES**
Simple smoke and Acceptance testing passed.

Anti-virus false positive report:
| File | Security vendor analysis |
|---|---|
| SLVersionChecker.exe | SecureAge - Malicious |
| slplugin.exe | MaxSecure - Trojan.Malware.300983.susgen<br> Microsoft - Trojan:Win32/Wacatac.B!ml<br> SecureAge - Malicious |

opsec-ee commented:
quiet cover .. I'm intrigued now.

kylelinden reviewed (approved):
2026.01 Beta approved
viewer = 26.1.0.21522948608
build = https://github.com/secondlife/viewer/releases/tag/Second_Life_Release%23b4a6af57-2026.01
cohort = 2026.01
desired = 500

Geenz commented:
deployed = https://github.com/secondlife/viewer/releases/tag/Second_Life_Release%23b4a6af57-2026.01

roxanneskelly committed:
In WebRTC.lib, Switch WASAPI playout to timer-driven mode with AUTOCONVERTPCM
  Use WAVEFORMATEXTENSIBLE and AUDCLNT_STREAMFLAGS_AUTOCONVERTPCM to
  support devices whose native format differs from 48kHz stereo PCM,
  such as Bluetooth HFP endpoints (16kHz mono) and surround sound
  devices (5.1/7.1).

  Replace the event-driven render loop (AUDCLNT_STREAMFLAGS_EVENTCALLBACK)
  with a timer-driven polling loop to avoid a known issue where
  AUTOCONVERTPCM combined with EVENTCALLBACK causes the audio engine to
  stop signaling render events, resulting in premature thread termination.

  Also adds 192kHz to the supported sample rate list, uses the first
  closest-match format from IsFormatSupported as a fallback when no
  exact match is found, and applies the same changes to the recording
  path.

akleshchev committed:
#5084 Adjust Window's watchdog to only run after login

like mainloop does

roxanneskelly committed:
Merge pull request #5421 from secondlife/roxie/webrtc-audio-dropout-fix

Fix bluetooth audio dropout issue

Dan-Linden commented:
| 2026.01 Beta 26.1.0.21999748351 | Result | Comment |
|---|---|---|
| secondlife/iqa#610 Acceptance, Integration & Additional Tests | Passed | Passed |
| **Overall Result** | **Passed** | **OK to ship** |

**NOTES**
Simple smoke and Acceptance testing passed.

Anti-virus false positive report:
| File | Security vendor analysis |
|---|---|
| SLVersionChecker.exe | SecureAge - Malicious |
| slplugin.exe | MaxSecure - Trojan.Malware.300983.susgen<br>  SecureAge - Malicious |

kylelinden reviewed (approved):
2026.01 Beta Approved
viewer = 26.1.0.21999748351
build = https://github.com/secondlife/viewer/releases/tag/Second_Life_Release%234db12c62-2026.01
cohort = 2026.01
desired = 1000

Geenz commented:
deployed = https://github.com/secondlife/viewer/releases/tag/Second_Life_Release%234db12c62-2026.01

akleshchev committed:
#5084 Adjust watchdog for bettet tracking of logout

Dan-Linden commented:
| 2026.01 Beta 26.1.0.22359044520 | Result | Comment |
|---|---|---|
| secondlife/iqa#614 Acceptance, Integration & Additional Tests | Passed | Passed |
| **Overall Result** | **Passed** | **OK to ship** |

**NOTES**
Simple smoke and Acceptance testing passed.


Anti-virus false positive report:
| File | Security vendor analysis |
|---|---|
| SLVersionChecker.exe | SecureAge - Malicious |
| slplugin.exe | MaxSecure - Trojan.Malware.300983.susgen<br> SecureAge - Malicious |

kylelinden reviewed (approved):
2026.01 Beta approved.
viewer = 26.1.0.22359044520
build = https://github.com/secondlife/viewer/releases/tag/Second_Life_Release%2344e21aa7-2026.01
cohort = 2026.01
desired = 3000

kylelinden commented:
Note: cohort manually increased to 2k around 9am 2/24/2026

Geenz commented:
deployed = https://github.com/secondlife/viewer/releases/tag/Second_Life_Release%2344e21aa7-2026.01

akleshchev committed:
#3612 Fix viewer not getting 'nonexistent region' responses

which are needed to process 'copy slurl'

akleshchev committed:
#5084 Disable watchdog in 26.1

There is a deadlock that looks to be caused by singleton+watchdog interactions, will resolve in 26.2

Dan-Linden commented:
| 2026.01 Beta 26.1.0.22641522367 | Result | Comment |
|---|---|---|
| secondlife/iqa#618 Acceptance, Integration & Additional Tests | Passed | Passed |
| **Overall Result** | **Passed** | **OK to ship** |

**NOTES**
Simple smoke and Acceptance testing passed.


Anti-virus false positive report:
| File | Security vendor analysis |
|---|---|
| SLVersionChecker.exe | SecureAge - Malicious |
| slplugin.exe | MaxSecure - Trojan.Malware.300983.susgen<br>Microsoft Trojan:Win32/Wacatac.B!ml<br> SecureAge - Malicious |

kylelinden reviewed (approved):
26.1 Beta approved
viewer = 26.1.0.22641522367
build = https://github.com/secondlife/viewer/releases/tag/Second_Life_Release%23e572093e-2026.01
cohort = 2026.01
desired = 3000

Geenz commented:
deployed = https://github.com/secondlife/viewer/releases/tag/Second_Life_Release%23e572093e-2026.01

kylelinden commented:
2026.01 approved for promotion to default release.
